docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Why We Created the Site Showcase
+date: 2018-06-15
+author: Shannon Soper
+tags: ["documentation", "launch"]
+---
+
+We recently finished the first phase of the [Site Showcase](https://next.gatsbyjs.org/showcase/), which is a searchable, filterable library of sites built with Gatsby! See a list of ways you can help in [Issue #5927](https://github.com/gatsbyjs/gatsby/issues/5927). Feel free to tackle any part of this project and contribute to it or discuss new possibilities!
+
+![Featured Sites](featured-sites.png)
+
+![Filtered Sites](filtered-sites.png)
+
+This is all possible because Cassie (@ThatOtherPerson) and Shawn (@sw-yx) put in a huge effort to get this developed, and Flo (@fk) did an amazing job on design. Many thanks to other people who contributed, including @gillkyle, @LeKoArts, @davidluhr, @daydream05, @m-allanson, @pieh, @calcsam and more (these are all GitHub names).
+
+## Submitting your site
+
+If you’d like to submit a site to the showcase, please follow the [instructions in this document](https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/site-showcase-submissions.md).
+
+## Why build a site showcase?
+
+We built the showcase to make it easier for you to (1) find the source code for open source Gatsby sites and (2) show off Gatsby to clients, coworkers ,friends & family.
+
+### Source code
+
+Sometimes, an example is worth 1000 words. That's why we added a GitHub icon next to sites with open source code.
```

Comment:
Is it part of phase two to add a way to filter sites by which ones are open source?